docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,294 @@
+
+### Camera (Visual Odometry, Visual SLAM)
+
+#### Expected situation
+
+- The vehicle is placed in an environment with rich visual features, such as an urban area.
+
+#### Situations that can make the system unstable
+
+- The vehicle is placed in a texture-less environment.
+- The vehicle is surrounded by other objects.
+- The camera observes significant illumination changes, such as those caused by sunshine, headlights from other vehicles or when approaching the exit of a tunnel.
+- The vehicle is placed in a dark environment.
+
+#### Functionality
+
+- The system can estimate odometry by tracking visual features.
+
+### Wheel speed sensor
+
+#### Expected situation
+
+- The vehicle is running on a flat and smooth road.
+
+#### Situations that can make the system unstable
+
+- The vehicle is running on a slippery or bumpy road, which can cause incorrect observations of wheel speed.
+
+#### Functionality
+
+- The system can acquire the vehicle velocity and estimate distance traveled.
+
+### IMU
+
+#### Expected environments
+
+- Flat, smooth roads
+
+#### Situations that can make the system unstable
+
+- IMUs have a bias[^1] that is dependent on the surrounding temperature, and can cause incorrect sensor observation or odometry drift.
+
+[^1]: For more details about bias, refer to the [VectorNav IMU specifications page](https://www.vectornav.com/resources/inertial-navigation-primer/specifications--and--error-budgets/specs-imuspecs).
+
+#### Functionality
+
+- The system can observe acceleration and angular velocity.
+- By integrating these observations, the system can estimate the local pose change and realize dead-reckoning
+
+### Geomagnetic sensor
+
+#### Expected situation
+
+- The vehicle is placed in an environment with low magnetic noise
+
+#### Situations that can make the system unstable
+
+- The vehicle is placed in an environment with high magnetic noise, such as one containing buildings or structures with reinforced steel or other materials that generate electromagnetic waves.
+
+#### Functionality
+
+- The system can estimate the vehicle's direction in the world coordinate system.
+
+### Magnetic markers
+
+#### Expected situation
+
+- The car is placed in an environment with magnetic markers installed.
+
+#### Situations where the system becomes unstable
+
+- The markers are not maintained.
+
+#### Functionality
+
+- Vehicle location can be obtained on the world coordinate by detecting the magnetic markers.
+- The system can work even if the road is covered with snow.
+
+## 3. Requirements
+
+- By implementing different modules, various sensor configurations and algorithms can be used.
+- The localization system can start pose estimation from an ambiguous initial location.
+- The system can produce a reliable initial location estimation.
+- The system can manage the state of the initial location estimation (uninitialized, initializable, or non-initializable) and can report to the error monitor.
```

Comment:
> (uninitialized, initializable, or non-initializable)

I feel it's not MECE (Mutually Exclusive and Collectively Exhaustive).

- `uninitialized` includes `initializable` and `non-initializable`.
- `initialized` is missing.

Reply:
So better to replace it with `(uninitialized or initialized)`?  @YamatoAndo @e-takeuchi

Reply:
I guess so.